lib/rake_migrations/config.rb

```diff
@@ -0,0 +1,27 @@
+module RakeMigrations
+  class Config
+    attr_accessor :database_name, :hostname
+
+    def initialize
```

Comment:
We might need to handle password protected database connections.
Generally, username and password combo is required to access such DB. PG connect would fail otherwise, exp ion production
[Blocker]

Reply:
Cool. I will make changes accordingly

Reply:
@yashBiz please check this out https://github.com/Bizongo/rake-migrations/pull/3